src/wifi/AdafruitIO_WINC1500.h

```diff
@@ -151,7 +151,7 @@ class AdafruitIO_WINC1500 : public AdafruitIO {
     @return   none
 */
 /**************************************************************************/
-    void _disconnect()
```

Comment:
Let's leave it as `_disconnect()` within the `wifi` classes so it matches the wifi interface's `connect` method.

Reply:
oops, misunderstood your request.

would it be better to simply add _mqtt->disconnect();  to the disconnect function?